### PR TITLE
Update transcripts

### PR DESCRIPTION
**Pull request by moonso**

### CODE REVIEW

**Comment by northwestwitch** [comment]
Since it's a change in the cli I've tried to install the branch on rasta stage by running (from rasta, hiseq.clinical):
`` update-scout-cli-stage.sh update-transcripts``

Something went wrong with genologics during the installation.

after this "us" command: I try to run:

``scout view --help``

And I get this: 
``
> 2018-10-12 15:35:59 rastapopoulos.scilifelab.se scout.commands.base[60555] INFO Running scout version 4.1.1
2018-10-12 15:35:59 rastapopoulos.scilifelab.se scout.commands.base[60555] INFO Setting database name to scout-stage
2018-10-12 15:35:59 rastapopoulos.scilifelab.se scout.adapter.client[60555] INFO Try to connect to mongodb://stageuser:****@localhost:27030/scout-stage
2018-10-12 15:35:59 rastapopoulos.scilifelab.se scout.adapter.client[60555] INFO Connection established
2018-10-12 15:35:59 rastapopoulos.scilifelab.se scout.commands.base[60555] INFO Setting up a mongo adapter
2018-10-12 15:35:59 rastapopoulos.scilifelab.se scout.commands.base[60555] INFO Check if authenticated...
Traceback (most recent call last):
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda//envs/stage/bin/scout", line 11, in <module>
    sys.exit(cli())
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/click/core.py", line 1063, in invoke
    Command.invoke(self, ctx)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/scout/commands/base.py", line 120, in cli
    for ins_obj in adapter.institutes():
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/pymongo/cursor.py", line 1176, in next
    if len(self.__data) or self._refresh():
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/pymongo/cursor.py", line 1070, in _refresh
    self.__session = self.__collection.database.client._ensure_session()
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1391, in _ensure_session
    return self.start_session(causal_consistency=False)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1369, in start_session
    server_session = self._get_server_session()
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1377, in _get_server_session
    return self._topology.get_server_session()
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/pymongo/topology.py", line 416, in get_server_session
    None)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/stage/lib/python3.6/site-packages/pymongo/topology.py", line 199, in _select_servers_loop
    self._error_message(selector))
pymongo.errors.ServerSelectionTimeoutError: No servers found yet
``

I don't know if the error has to do with my inability to understand and remember all the commands for setting up the testing environment or the code itself, but I'm leaving it to the next week because it's getting very frustrating, sorry!

**Comment by moonso** [comment]
Hi, I think that we need to merge #1009 before we can run the cli from rasta

**Comment by moonso** [comment]
Hi @northwestwitch and @dnil ! I would really like to merge this, it's a smack thing. You could try in your local installation if you want to test it.

**Comment by northwestwitch** [comment]
I'll try on my local installation. Need to fix a thing before tho

**Comment by moonso** [comment]
The problem with the demo instance is that it's missing transcripts. It is a rather large operation for me to fix that at the moment. If you run `scout setup database` and then try the command on the `scout` database it will work.

File: scout/commands/view/transcripts.py [hunk 1]
```diff
@@ -7,14 +7,20 @@
...
-def transcripts(context, build):
+def transcripts(context, build, hgnc_id, json):
     """Show all transcripts in the database"""
     LOG.info("Running scout view transcripts")
     adapter = context.obj['adapter']
 
-    click.echo("Chromosom\tstart\tend\ttranscript_id\thgnc_id\trefseq\tis_primary")
-    for tx_obj in adapter.transcripts(build=build):
+    if not json:
+        click.echo("Chromosom\tstart\tend\ttranscript_id\thgnc_id\trefseq\tis_primary")
```
Comment (northwestwitch) [inline scout/commands/view/transcripts.py]: ChromosomE here

**Comment by northwestwitch** [comment]
Tested and it works, both the table and the json format!

**Comment by northwestwitch** [comment]
I approve this not so I avoid that you bug me while on the train @moonso, but fix the header! :)

**Pull request merged**